packages/cbioportal-frontend-commons/src/components/SVGAxis.tsx

```diff
@@ -146,6 +146,24 @@ export default class SVGAxis extends React.Component<SVGAxisProps, {}> {
                         : horizontalPadding);
                 transform = '';
             }
+            let wrappedLabel;
+            if (this.props.label.endsWith('Mutations')) {
+                wrappedLabel = (
```

Comment:
what's reason for this?    we should evaluate whether there's some more generalized way to handle this.  also, make sure to put a comment on code like

Reply:
remind me to discuss this with you again.  we have to come up with a better solution.